README.md

```diff
@@ -134,77 +138,130 @@ ROOT
 
 ## Package structure validation rules
 The package structure is validated according to the following rules:
+
 * Required XML and XSD files exist in the package and are well-formed:
-  * arkivstruktur.xsd, endringslogg.xsd, offentligJournal.xsd, loependeJournal.xsd, addml.xsd, metadatakatalog.xsd
-  * arkivstruktur.xml, endringslogg.xml, offentligJournal.xml, loependeJournal.xml, arkivuttrekk.xml
+  * arkivstruktur.xsd, endringslogg.xsd, offentligJournal.xsd, loependeJournal.xsd, addml.xsd, metadatakatalog.xsd, where offentligJournal.xsd and loependeJournal.xsd are optional
+  * arkivstruktur.xml, endringslogg.xml, offentligJournal.xml, loependeJournal.xml, arkivuttrekk.xml, where offentligJournal.xml and loependeJournal.xml are optional
 * The checksums of all XSD schemas in the package match the corresponding checksums of the schemas distributed with the Noark 5 standard
 * All XML files in the package match the corresponding XSD schemas distributed with the Noark 5 standard
 
 ## Content validation rules
+There are two distinct categories of validation rules - Checks and Tests. Both of them are identified by an ID, a title, a description, and a group. Checks, however, may only include information about a specific aspect of an extraction package whereas Tests provide additional support for generating warnings and/or errors. Generally, you will see checks printed before tests in produced validation reports.
```

Comment:
`Checks produce information about a specific aspect of an extraction package whereas Tests perform validations and may produce warnings and errors.`